docs/releasing.md

```diff
@@ -39,6 +39,15 @@
 
    - Use draft created by `release` GitHub workflow.
    - Use the [CHANGELOG.md](../CHANGELOG.md) content in the description.
+   - Add tests section in description displaying current status of testing:
+
+    ```markdown
+   ## Tests
+
+   - [ ] CI
+   - [ ] MacOS with Linux Containers
+   - [ ] Windows with Linux Containers
+    ```
```

Comment:
This should be in the previous step just after "After you've pushed the git tag, a `release` GitHub workflow starts. This will create draft release with uploaded artifacts."

Here you are already making a GH release.